### PR TITLE
Various key binding extensions

### PR DESCRIPTION
**Pull request by stuartleeks**
* Remove the handler.Index in favour of just the string IDs
* Allow keys such as '`'` `k` to be bound to (as well as the previous `Ctrl+J` etc)
* Allow multiple bindings for keys (e.g. binding `ListUp` to `h` and `Up`)

### CODE REVIEW

File: internal/pkg/keybindings/listhandlers.go [hunk 1]
```diff
@@ -590,3 +590,28 @@ func getAPIErrorMessage(responseString string) (string, error) {
...
+		List:   list,
+		status: status,
+	}
+	handler.id = HandlerID_ListLetterS
+	return handler
+}
+
+func (h ListLetterSHandler) Fn() func(g *gocui.Gui, v *gocui.View) error {
+	return func(g *gocui.Gui, v *gocui.View) error {
+		h.status.Status("S pressed", false)
```
Comment (lawrencegripper) [inline internal/pkg/keybindings/listhandlers.go]: Guessing this one is for testing?

**Comment by lawrencegripper** [comment]
@stuartleeks first up - THIS IS AWESOME very excited to get this merged. 

second up, likely won't get time to review today and I'm holiday next week but will pickup first thing next week if that's ok?

File: internal/pkg/keybindings/listhandlers.go (hunk 1 above)
Comment (stuartleeks) [reply internal/pkg/keybindings/listhandlers.go]: Nope, just leftover code that I didn't clean up!

**Comment by stuartleeks** [comment]
No problem - will get on to tidying up as soon as I can

**Comment by stuartleeks** [comment]
ok - updated to remove dud code and get build passing :-)

**Pull request merged**